released/packages/coq-bignums/coq-bignums.8.8.0/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "1.2"
+maintainer: "pierre.letouzey@inria.fr"
```

Comment:
Hum, this field is for the maintainer of the opam package, but you are not Pierre.
Please update this info.

Reply:
I think @thery  is a good value for this field since he is the one that packaged the 8.7 version (and put the wrong metadata in place). Unless you want to take over the package.